geotools-suite-8.4/pom.xml

```diff
@@ -78,7 +78,7 @@
 		<dependency>
 			<groupId>org.geotools</groupId>
 			<artifactId>gt-epsg-hsql</artifactId>
-			<version>${bundle.version}</version>
+			<version>13.1</version>
```

Comment:
We sure this version (13.1) is compatible with the old one (8.4)? This looks like a huge jump. Couldn't we move everything to 13.1 instead?